packages/edit-site/src/components/dataviews/add-filter.js

```diff
@@ -0,0 +1,70 @@
+/**
+ * WordPress dependencies
+ */
+import {
+	privateApis as componentsPrivateApis,
+	Icon,
+	Button,
+} from '@wordpress/components';
+import { chevronDown, check } from '@wordpress/icons';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import { unlock } from '../../lock-unlock';
+
+const { DropdownMenuV2, DropdownMenuItemV2 } = unlock( componentsPrivateApis );
+
+export default function AddFilter( { dataView, filters, onChangeFilters } ) {
+	const filterableFields = dataView
```

Comment:
We should avoid using `dataView` (the tan stack APIs) here, instead we should use the `fields` prop. The reason is simple, filters are cross view type (layout), and tan stack is specific to the table.